build/index.js

```diff
@@ -103,6 +103,18 @@ function injectLoadingLazyAttributes($) {
   $("img:not([loading]), iframe:not([loading])").attr("loading", "lazy");
 }
 
+/**
+ * Find all `in-page-callout` div elements and rewrite
+ * to be just `callout`, no more need to mark them as `webdev`
+ * @param {Cheerio document instance} $
+ */
+function rewriteInPageCallout($) {
+  $("div.in-page-callout").each((i, element) => {
+    $(element).removeClass("in-page-callout webdev");
+    $(element).addClass("callout");
+  });
```

Comment:
No need to loop. You can do what the other ones are doing. So
```javascript
$("div.in-page-callout").removeClass("in-page-callout webdev").addClass("callout");
```
plus whatever Prettier has to say. 
And perhaps to stick with the convention of naming it `injectInPageCallout`.